documentation-website/Writerside/snippets/exposed-dao/src/main/kotlin/org/example/entities/EntityWithBase64.kt

```diff
@@ -4,12 +4,15 @@ import org.jetbrains.exposed.dao.IntEntity
 import org.jetbrains.exposed.dao.IntEntityClass
 import org.jetbrains.exposed.dao.id.EntityID
 import org.jetbrains.exposed.dao.id.IntIdTable
+import kotlin.io.encoding.Base64
+import kotlin.io.encoding.ExperimentalEncodingApi
 
 object TableWithText : IntIdTable() {
     val text = varchar("text", length = 2048)
 }
 
 class EntityWithBase64(id: EntityID<Int>) : IntEntity(id) {
+    @OptIn(ExperimentalEncodingApi::class)
```

Comment:
`Base64` is still experimental, so this opt-in is required to compile. If you think it's better to avoid this in a sample code block, I could try opting in at the build file level, or maybe using `java.util.Base64` instead, or refactoring the example entirely.

Reply:
If it won't overcomplicate the example, then it might be nice to show a more stable alternative.

Reply:
Ok I switched to the stable java object to keep the meaning of the example.